arlo_server/routes.py

```diff
@@ -394,9 +401,15 @@ def update_jurisdictions_file(election_id=None):
     jurisdictions_file = request.files["jurisdictions"]
     jurisdictions_file_string = jurisdictions_file.read().decode("utf-8-sig")
 
-    election.jurisdictions_file = jurisdictions_file_string
-    election.jurisdictions_filename = jurisdictions_file.filename
-    election.jurisdictions_file_uploaded_at = datetime.datetime.utcnow()
+    if election.jurisdictions_file:
+        db.session.delete(election.jurisdictions_file)
```

Comment:
do you have a test for overwriting?

Reply:
I do here: https://github.com/votingworks/arlo/pull/356/files#diff-0e2be5baf85cbad5135cb3e8ad217724R151